lang/swi-prolog-devel/Portfile

```diff
@@ -2,7 +2,6 @@
 
 PortSystem          1.0
 PortGroup           cmake 1.1
-PortGroup           compiler_blacklist_versions 1.0
```

Comment:
that is okay, but then you will need remove the lines (38-39 or so) where you do the compiler.blacklist stuff as well.